src/view/uniprotkb/components/OrganismView.tsx

```diff
@@ -58,9 +59,12 @@ export const OrganismListView: React.FC<{
     infoListData.push({
       title: 'Organism',
       content: (
-        <Link to={`/taxonomy/${data.taxonId}`}>
-          {`${data.scientificName} (${data.commonName})`}
-        </Link>
+        <Fragment>
+          <Link to={`/taxonomy/${data.taxonId}`}>
+            {`${data.scientificName} (${data.commonName})`}
+          </Link>
+          {data.evidences && <UniProtEvidenceTag evidences={data.evidences} />}
```

Comment:
If `data.evidences = []` this will return an empty fragment. Probably doesn't matter but could equally consider the length too: `data.evidences && data.evidences.length && ...`